lib/src/component.dart

```diff
@@ -51,10 +98,20 @@ abstract class FluxComponent<ActionsT, StoresT> extends react.Component {
     }
   }
 
+  /// If you need more fine-grained control over listening to stores, override
+  /// this method to return a Map of stores to handlers. Whenever a store in
+  /// the returned map triggers, the respective handler will be called.
+  ///
+  /// If possible, [redrawOn] should be used instead of this in order to avoid
```

Comment:
should probably note that `getStoreHandlers` have precedence over `redrawOn` handling